kbase-extension/scss/all_concat.scss

```diff
@@ -32,6 +32,7 @@
 @import "partials/editorCell";
 
 // order should not be important for the following files
+@import "partials/appInput";
```

Comment:
New one! This will be a home for other specific widget things later (remember those inline css bits in intInput, floatInput, etc?).